File: Codechef/Practice/Learning SQL.go
https://www.codechef.com/practice/course/go/PGOL02/problems/LEARNSQL




package main

import (
    "fmt"
)

func main() {
    var R, C, E int
    fmt.Scan(&R, &C, &E)
    
    // Total rows after adding extra rows
    totalRows := R + E
    
    // Total number of cells
    totalCells := totalRows * C
    
    fmt.Println(totalCells)
}

File: Codechef/Practice/Learning SQL.js
https://www.codechef.com/practice/course/javascript/PRACTICEJS01/problems/LEARNSQL




// Complete the code

function totalCells(r, c, e){
   console.log((r+e)*c);
}

// Input related code. Please do not change this.
process.stdin.setEncoding('utf8');
process.stdin.on('data', function(input) {
  const nums = input.trim().split(' ');
  const r = parseInt(nums[0]); 
  const c = parseInt(nums[1]); 
  const e = parseInt(nums[2]); 
  totalCells(r, c, e);
});